README.md

```diff
@@ -34,10 +34,6 @@ Clone this repo
 1. `npm install`
 2. `npm test`
 
-### 4. Before committing
-
-`npm run build` (and commit the changes to lib/)
```

Comment:
@pscohn it was previously getting build to build/main.js (for the browser) *and* lib/*.js (as a node module). The module was really just for the fake plugin on schrute. This PR just drops the lib/ stuff; schrute fake plugin will just import the browser version (https://github.com/rainforestapp/schrute/pull/54).